### ISSUE TITLE
Version Does Not Print Anything

### ISSUE THREAD

**Issue by nisdas** [prysmaticlabs/prysm#3895]
If we run `./beacon-node --version`, we get 
```
2019/10/30 13:38:35 maxprocs: Leaving GOMAXPROCS=8: CPU quota undefined
beacon-chain version Git commit: {STABLE_GIT_COMMIT}. Built at: {DATE}
```
which does not tell us anything, as the git commit and date are both empty. This issue is to track the implementation of the proper version information of the beacon node being printed.

**Comment by mkinney** [comment]
The code is in `shared/version/version.go`.

**Comment by mkinney** [comment]
The script does the work:

$ cat scripts/workspace_status.sh

```
#!/bin/bash

echo STABLE_GIT_COMMIT $(git rev-parse HEAD)
echo DATE $(date --rfc-3339=seconds --utc)
echo DOCKER_TAG $(git rev-parse --abbrev-ref HEAD)-$(git rev-parse --short=6 HEAD)
```

But when I run it...

```./scripts/workspace_status.sh
STABLE_GIT_COMMIT 6909ebe2855d5b0d450b3dee40e62a54c273eeff
date: illegal option -- -
usage: date [-jnRu] [-d dst] [-r seconds] [-t west] [-v[+|-]val[ymwdHMS]] ...
            [-f fmt date | [[[mm]dd]HH]MM[[cc]yy][.ss]] [+format]
DATE
DOCKER_TAG add_arg_to_clear_db-6909eb
```

Looks like a mac/date issue.

**Comment by prestonvanloon** [comment]
You have to build the beacon chain node with the stamp.

```
bazel build --define ssz=minimal //beacon-chain --workspace_status_command=./scripts/workspace_status.sh --stamp
```

The version could be better about detecting that the binary was not stamped and print a better message.

**Comment by rauljordan** [comment]
Resolved by #3904
